### DISCUSSION TITLE
Image information should include File size

### DISCUSSION THREAD

**Discussion by KIRSJ** [community/community#55464] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I experienced a situation where it would have been nice to have File size at hand in Image information.
I was a bit surprised when after a closer examination I realized that this wasn't there. I wonder why such an obvious information so far has been excluded.
It's easy to point at different other products where File size 'naturally' is at hand when image information is listed.
This topic was briefly discussed in pixls.us months ago without gaining much interest - and my recent attempt to deliver a revival has gloriously failed.
So, now I hope this field can be added if the process is not extraordinary complex compared to the gain.

**Comment by Flako135** [comment]
[The+Owner's+Rewards+Card+by+M1+Terms+&+Conditions.pdf](https://github.com/community/community/files/11507668/The%2BOwner.s%2BRewards%2BCard%2Bby%2BM1%2BTerms%2B.%2BConditions.pdf)
